wagtail/wagtailembeds/format.py

```diff
@@ -22,16 +22,20 @@ def embed_to_frontend_html(url):
             })
         else:
             return ''
-    except:
+    except embeds.EmbedNotFoundException:
         return ''
 
 
 def embed_to_editor_html(url):
```

Comment:
Returns two types when error conditions are met `str` and `NoneType`. Standardise on one?